Src/FluentAssertions/Equivalency/IEquivalencyAssertionOptions.cs

```diff
@@ -71,6 +73,12 @@ public interface IEquivalencyAssertionOptions
         /// </summary>
         MemberVisibility IncludedFields { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether members marked with [EditorBrowsable]
```

Comment:
🔧 Would be nice to use `<see` constructs when referring to types and members.

Reply:
Easy enough to do. :-)